packages/webchat/src/main.tsx

```diff
@@ -168,12 +190,15 @@ class Web extends React.Component<MainProps> {
     })
 
     observe(this.props.config!, 'overrides', (data: any) => {
+      // TODO: 12 - we didn't check if window.parent was set the first time
+      // we loaded overrideds, so why do we do now?
       if (data.newValue && window.parent) {
         this.loadOverrides(data.newValue)
```

Comment:
I think that was only if you changed the config on the emulator to see the change in real time